src/strings/csv/es.csv

```diff
@@ -386,6 +386,9 @@ GRAMINOID,Gramínea
 GRAMS,Gramos
 GREATER_THAN_FIFTY_PERCENT,> 50%
 GROWTH_FORM,Forma de crecimiento
+HIGHEST,Más alta
+HIGHEST_AND_LOWEST_MORTALITY_RATE_CARD_TITLE,¿Qué zonas tiene la tasa de mortalidad más alta y más baja?
+HIGHEST_AND_LOWEST_MORTALITY_RATE_SPECIES_CARD_TITLE,¿Qué especie tiene la tasa de mortalidad más alta y cuál la más baja?
```

Comment:
@constanzauanini Alex fixed this seems like. We just need to revert the other one.

Reply:
This one is correct! 💯